### DISCUSSION TITLE
No response to my comments in your reddit thread

### DISCUSSION THREAD

**Discussion by metaperl** [causalens/dara#58] [General]
In [your recent reddit thread](https://www.reddit.com/r/Python/comments/1614om5/comment/jxt4sii/?utm_source=share&utm_medium=web2x&context=3) you stated:

>  Don't forget to leave us feedback on GitHub if you check out the framework, we're always looking to improve it.

I left a number of comments in that thread thread and none have been answered. This is a very impressive offering and I am sure it will do well with or without me ever mentioning it in [my survey of pure python web framework solutions](https://metaperl.github.io/pure-python-web-development/intro.html).

That being said, it will be listed as Class B until it meets [Class A  requirements](https://metaperl.github.io/pure-python-web-development/practical-guide.html).

I hope the tone of my posts was acceptable in your reddit thread.

**Comment by krzysztof-causalens** [comment]
Hey,
Apologies for not responding sooner, we simply missed your comment. Could you please let us know   which particular questions would you like us to answer? We did try to respond to every question in the reddit thread at the time. I can see there are a few more recent replies which we'll respond to shortly.

Could you also pleases let us know which Class A requirements do you think we are not meeting? I can see you mentioned the lack of examples of session and authorization in your entry about Dara, which is fair - we will update our documentation to showcase those capabilities.

**Reply by metaperl** [reply]
Thank you for replying.

> I can see you mentioned the lack of examples of session and authorization in your entry about Dara, which is fair - we will update our documentation to showcase those capabilities.

Ok looking forward to the update.
